examples/demo/src/index.ts

```diff
@@ -1,14 +1,41 @@
 import './index.css';
-import { editor, Environment, languages } from 'monaco-editor';
+import { parse } from 'jsonc-parser';
+import { editor, Environment, languages, Uri } from 'monaco-editor';
 import { configureMonacoTailwindcss, tailwindcssData } from 'monaco-tailwindcss';
+import { TailwindConfig } from 'tailwindcss/tailwind-config';
 
 declare global {
   interface Window {
     MonacoEnvironment: Environment;
   }
 }
 
-configureMonacoTailwindcss();
+const initialConfig: TailwindConfig = {
+  theme: {
+    extend: {
```

Comment:
```suggestion
    extend: {
        "screens": {
            "television": "90000px",
        },
        "spacing": {
            "128": "32rem"
        },
```

as this works too i think we should add that so we can tests that custom breakpoints work fx.